Serhii_Oliinyk/3/poker.py

```diff
@@ -0,0 +1,309 @@
+def check_is_royal_flush(item):
+    """Function check if royal flush.
+
+    Args:
+        item: type str.
+
+    Returns:
+        Return True or False.
+
+    """
+
+    hand_number = []
+    hand_suit = []
+
+    phrases = item.split(' ')
+
+    for i in phrases:
+        number = i[:len(i) - 1]
+        hand_number.append(number)
+
+        suit = i[len(i) - 1:]
+        hand_suit.append(suit)
+
+    result_number = False
+
+    if (("A" in hand_number) and
```

Comment:
You can create a collection of those cards and just check it using that collection.